net-dns/dnssec-validator/dnssec-validator-2.2.3.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+MY_PN="dnssec-tools"
+MY_PN2="DNSSEC-Tools"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Tools to ease the validation of DNSSEC related technologies"
+HOMEPAGE="https://www.dnssec-tools.org/"
+SRC_URI="https://github.com/${MY_PN2}/${MY_PN2}/archive/${MY_P}.tar.gz"
+
+KEYWORDS="~amd64 ~arm ~x86"
+LICENSE="BSD"
+SLOT="0"
+IUSE="dlv +ipv6 +nsec3 +threads"
+
+RDEPEND=">dev-libs/openssl-1.0.2:0"
```

Comment:
`>=`?

Reply:
See @robbat2 comment. It won't compile on ```openssl-1.0.2``` series.

Reply:
Then you want `>=1.1.0`

Reply:
Done.